compiler/circle-quantizer/requires.cmake

```diff
@@ -5,3 +5,4 @@ require("luci")
 require("oops")
 require("arser")
 require("vconone")
+require("record-hessian")
```

Comment:
plz add new lines so that ![image](https://github.com/user-attachments/assets/dcaf88e1-4742-4316-817e-b50f329dfcef) is not shown